README.md

```diff
@@ -0,0 +1,23 @@
+# Vending Machine
+
+## Version: 1.0.0
+
+### Description:
+vending machine designed by means of matrixes, 

```

Comment:
Capitalize the Vending and the first letter of each paragraph.

Reply:
Done